client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -269,6 +269,17 @@ interface ChatClient {
         channelExtraData: Map<String, Any> = emptyMap()
     ): Call<Channel>
 
+    fun enableSlowMode(
+        channelType: String,
+        channelId: String,
+        cooldownTimeInSeconds: Int
```

Comment:
Just a quick thought, we could add a default value to this if we want. Don't know if we have one in the other APIs.

Reply:
Reading the docs it seems that there aren't any default value for that.
The only limitation is that couldown time should be between 0 and 120, but it is handled by backend, in the future it could be a configurable attribute that every customer configures on the dashboard